recipe/meta.yaml

```diff
@@ -46,33 +37,28 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     # pythran code needs clang-cl on windows
-    - clang                           # [win]
+    - clang   # [win]
+    - lld     # [win]
     - {{ compiler('fortran') }}
-    - patch         # [s390x]
-    - m2-patch      # [win]
-    - pkg-config    # [osx]
+    - patch   # [s390x]
+    - pkg-config
   host:
     - python
-    - cython 0.29.32
-    - pybind11 2.10.1
-    - pythran 0.12.1
-    - setuptools <60
-    - wheel <0.39.0
+    - cython >=0.29.35,<3.0
```

Comment:
should not we have strict pinning ( to specific version ) in host section rather than range ?

Reply:
I could pin to 0.29.* as there is no good reason not to use the latest minor. But then the upstream range is more precise than this, so I chose to use it.